libdokan/fs.go

```diff
@@ -377,11 +377,11 @@ func (f *FS) open(ctx context.Context, oc *openContext, ps []string) (dokan.File
 			return nil, false, dokan.ErrAccessDenied
 		}
 		return f.root.private.open(ctx, oc, ps[1:])
-	case `TEAM` == ps[0]:
+	case strings.ToUpper(TeamName) == ps[0]:
```

Comment:
Dunno!  I was just making it consistent with the other places above.  I think this one happened concurrently with switchover to `ToUpper`.  I'm not worried about performance here, but if you are we can revert to using constants in a separate PR.